primitives/election-providers/src/lib.rs

```diff
@@ -160,23 +163,43 @@
 
 pub mod onchain;
 use sp_std::{prelude::*, fmt::Debug};
+use data_provider::ReturnValue;
 
 /// Re-export some type as they are used in the interface.
 pub use sp_arithmetic::PerThing;
 pub use sp_npos_elections::{Assignment, ExtendedBalance, PerThing128, Supports, VoteWeight};
 
+pub mod data_provider {
+	/// Alias for the return value of the election data provider.
+	pub type ReturnValue<T, Aux> = Result<(T, Aux)>;
```

Comment:
I think you can directly write: `Result<(T, Aux)>` instead of `ReturnValue<T, Aux>`, the former is more understandable, `ReturnValue` doesn't even tell it is a result.

Reply:
Yeah true, it is even longer.

Reply:
(Now I see that intention was to have it named something other than result to avoid conflict, but I agree this is not idiomatic)